crates/re_viewer_context/src/lib.rs

```diff
@@ -114,6 +118,10 @@ impl SpaceViewId {
     pub fn gpu_readback_id(self) -> re_renderer::GpuReadbackIdentifier {
         re_log_types::hash::Hash64::hash(self).hash64()
     }
+
+    pub fn as_entity_path(&self) -> EntityPath {
+        EntityPath::from(format!("{}/{}", Self::SPACEVIEW_PREFIX, self))
```

Comment:
[thought process, not actionable] here and with with the `/properties` paths I'm wondering if we can add some special syntax that makes this more uniquely identifiable as a built-in thing. Some character that we don't allow for regular logging etc. 🤔. We're essentially establishing "magic paths" now. Not necessarily a bad thing

Reply:
On one-hand, yes they're magic, but only within the context of the blueprint store.

That said, I totally agree that having a blueprint path-schema constants and validator will make this all much less error prone.